cmsplugin_articles_ai/cms_plugins.py

```diff
@@ -13,12 +13,14 @@ class ArticleList(CMSPluginBase):
     module = _("Articles")
     name = _("List of latest articles")
     render_template = "cmsplugin_articles_ai/article_lift_list.html"
-    fields = ["article_amount", "language_filter"]
+    fields = ["article_amount", "language_filter", "exclude_categories"]
 
     def get_articles(self, plugin_conf):
         return Article.objects.public(
             language=plugin_conf.language_filter,
-        ).filter(publisher_is_draft=get_draft_status())
+        ).filter(
+            publisher_is_draft=get_draft_status()
```

Comment:
Wondering why `public` doesn't already filter the drafts away. Should we fix that too?

Reply:
I'd say out of scope here, though I was wondering the same. Maybe it's something to do with the editor/non-editor view?